src/manifest.ts

```diff
@@ -79,6 +79,16 @@ export namespace Commands {
         icon: 'desktop-download',
         title: 'Export All'
     } as const;
+    export const IGNORE_PERIPHERAL: CommandDefinition = {
```

Comment:
Just curious: why do we all including icons and title here? Wouldn't definitions in `package.json` and the `commandId` here be sufficient?

Reply:
Yep, it should be sufficient. I will fix that.